### ISSUE TITLE
keyboard layout with variant apostrophe not work

### ISSUE THREAD

**Issue by agrisvv** [neutrinolabs/xrdp#1027]
Fedora 27  with xrdp 0.9.5 remote keyboard input layout Latvia lv: Apostrophe (') variant

Problem then press '+a:
   'a'a'a''''ā'ā'ā'ā'ā'ā'ā'ā'ā'ā'ā''''ā'ā'ā' 'e'e'e'ē'ē''ē''ē'
Correct:
   āāāāāāāāā

 Apostophe(') need to show only when press twice. No problems when use right ALT ( differnet keyboard variant), but with than not so easy to write.
 1. local - correct.
 1. vnc client to same session all ok. - correct
 2. windows or linux remmina rdp protocol. - not correct
 
 setxkbmap -print -verbose 10
```
Setting verbose level to 10
locale is C
Trying to load rules file ./rules/evdev...
Trying to load rules file /usr/share/X11/xkb/rules/evdev...
Success.
Applied rules from evdev:
rules:      evdev
model:      pc105
layout:     lv,us
variant:    apostrophe,
Trying to build keymap using the following components:
keycodes:   evdev+aliases(qwerty)
types:      complete
compat:     complete
symbols:    pc+lv(apostrophe)+us:2+inet(evdev)
geometry:   pc(pc105)
xkb_keymap {
	xkb_keycodes  { include "evdev+aliases(qwerty)"	};
	xkb_types     { include "complete"	};
	xkb_compat    { include "complete"	};
	xkb_symbols   { include "pc+lv(apostrophe)+us:2+inet(evdev)"	};
	xkb_geometry  { include "pc(pc105)"	};
```

**Comment by agrisvv** [comment]
more tests:
1. On client use keyboard US in remmina. on server latvian  apostrophe. ( result feels like two keyboards write at same time when write chars with LEVEL3 on (') key.
2. Generate key map km-00000426.ini from fedora 27 local also try from ubuntu 16.04 (not help). Copy to server and write config in xrdp_keyboard.ini. From logs can see whats ini are picked up.

But now when press LEVEL3 (') key sometimes writes correct (ā) sometimes stuck and write nothing. Only after presing other keys without LEVEL3 char, i get out of stuck.

**Comment by matt335672** [comment]
I'm closing this (neglected) issue, as we've now merged #3138 which should make this stuff a lot easier to debug and sort out in the future.
